$NetBSD$

Adjust setup.py to put documention in a reasonable place rather than
in $prefix directly.
Filed upstream as
  http://trac-hacks.org/ticket/6768

--- setup.py.orig	2010-03-04 19:35:00.000000000 +0000
+++ setup.py
@@ -20,4 +20,4 @@ setup(
     packages=['tracext', 'tracext.git'],
     namespace_packages=['tracext'],
     entry_points = {'trac.plugins': 'git = tracext.git.git_fs'},
-    data_files=['COPYING','README'])
+    data_files=[('share/doc/trac-plugins-GitPlugin', ['COPYING','README'])])
